### ISSUE TITLE
Something is wrong with the horizontal symmetry axis

### ISSUE THREAD

**Issue by leon-vv** [PyAbel/PyAbel#285]
```
import numpy as np
import abel

xx, yy = np.meshgrid(np.linspace(-256, 256, 512), np.linspace(-256, 256, 512))
r = np.sqrt(xx**2 + yy**2)

a = np.exp(-1/5000*r**2)

plt.imshow(a)
plt.colorbar()

t = abel.Transform(a, symmetry_axis=0).transform;

plt.figure()
plt.imshow(t)
```

This results in

![image](https://user-images.githubusercontent.com/4085287/80280891-78d57f80-8707-11ea-80d9-dd38c02446a7.png)

which looks just fine.

Now keeping with the vertical axis of symmetry I move the gauss function along the y axis.
`r = np.sqrt(xx**2 + (yy-80)**2)`. The result is as expected:

![image](https://user-images.githubusercontent.com/4085287/80280932-bd611b00-8707-11ea-826c-737f783576d8.png)

Now move the gauss along the x-axis  `r = np.sqrt((xx-80)**2 + yy**2)` and choose a horizontal axis of symmetry: `t = abel.Transform(a, symmetry_axis=1).transform;`. This should result in the same images, except transposed. But we get:

![image](https://user-images.githubusercontent.com/4085287/80281010-6ad42e80-8708-11ea-83ba-8a301d2b9283.png)

**Comment by stggh** [comment]
@leon-vv thanks for providing the code to illustrate this issue.

Your second case breaks cylindrical symmetry and so the Abel transform is invalid. See the [docs](https://pyabel.readthedocs.io/en/latest/readme_link.html), and the wine glass image figure.

In your example, specifying the symmetry origin facilitates the Abel transform. Passing the `origin` (centre of symmetry) parameter for the transform fixes this. Note, also, the image width should be an odd integer for `PyAbel` transforms.
```python
xx, yy = np.meshgrid(np.linspace(-255, 255, 511), np.linspace(-255, 255, 511))
r = np.sqrt((xx-80)**2 + yy**2)

a = np.exp(-1/5000*r**2)

plt.imshow(a, extent=[-255, 255, -255, 255])
plt.colorbar()

t = abel.Transform(a, symmetry_axis=1, origin=(255, 255+80)).transform

plt.figure()
plt.imshow(t, extent=[-255, 255, -255, 255])

plt.show()
```
![image](https://user-images.githubusercontent.com/10932229/80294474-7aa54e80-87ac-11ea-8290-300c20c6d6e1.png)

**Comment by DanHickstein** [comment]
I think that @leon-vv is interpreting the `symmetry_axis` argument as a way to control the relevant axis for the Abel transform. Really, the Abel transform is always performed around the y-axis. I think that we should make this clear in the docs for [abel.Transform](https://pyabel.readthedocs.io/en/latest/abel.html#module-abel.transform). I'm noticing a few issues with the doc for our flagship user interface:

1. We should let users know that the Abel transform is performed around the y-axis. If their data is rotated the wrong way, they are going to need to use `np.rot90`.

2. In the description for "symmetry_axis", we should probably add a bit more info. At least we should say "see below", since we provide a better description for quadrants and symmetry below. 

3. We should provide an usage example! (I know that we have one in the readme, maybe we just cut and paste it here.)

And now, a side note. @stggh said:
> Note, also, the image width should be an odd integer for PyAbel transforms.

`abel.Transform` takes care of re-centering and can deal with even image widths. So, while the "odd image requirement" is true on a fundamental level for each transform method, `abel.Transform` takes care of this issue and allows both even and odd images to be used. So, we should simply encourage users to use abel.Transform and live a care-free life :).  (I'm thinking back to  https://github.com/PyAbel/PyAbel/issues/272, we had a user asking about the even/odd image situation, when really they were just looking for the centering + transform functionality of `abel.Transform`.)

**Comment by stggh** [comment]
> `abel.Transform` takes care of re-centering and can deal with even image widths

Only when the `origin` parameter is specified.

**Comment by MikhailRyazanov** [comment]
> We should let users know that the Abel transform is performed around the y-axis. If their data is rotated the wrong way, they are going to need to use `np.rot90`.

Or transpose before and after. May be faster or may be not, but at least `(A.T).T == A`.

**Comment by leon-vv** [comment]
Thanks all for your elaborate answers.

> Your second case breaks cylindrical symmetry and so the Abel transform is invalid.

> I think that @leon-vv is interpreting the symmetry_axis argument as a way to control the relevant axis for the Abel transform. 

Yes this is the root of the problem. I thought the symmetry_axis determined the z-axis of the cylindrical coordinate system used for the abel transform.

> Or transpose before and after. May be faster or may be not, but at least `(A.T).T == A`

This is what I have resorted to. `np.rot90` uses a transpose under the hood, so directly using transpose will be faster.
 
> We should let users know that the Abel transform is performed around the y-axis. If their data is rotated the wrong way, they are going to need to use np.rot90.
> 
> In the description for "symmetry_axis", we should probably add a bit more info. At least we should say "see below", since we provide a better description for quadrants and symmetry below.

Please see my pull request.

**Comment by MikhailRyazanov** [comment]
Should we close this issue, since the problem is already addressed in #288?

**Comment by DanHickstein** [comment]
Yes, I think that this has been addressed. So I'll close this. (If someone considers this unresolved, please re-open.)

Thanks for pointing this out @leon-vv!
